src/qs8-vadd/gen/qs8-vadd-minmax-hvx-u128.c

```diff
@@ -0,0 +1,182 @@
+// Auto-generated file. Do not edit!
+//   Template: src/qs8-vadd/hvx.c.in
+//   Generator: tools/xngen
+//
+// Copyright 2024 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <assert.h>
+
+#include <hvx_hexagon_protos.h>
+#include <hexagon_protos.h>
+#include <hexagon_types.h>
+
+#include <xnnpack/vbinary.h>
+#include <xnnpack/intrinsics-polyfill.h>
+
+
+void xnn_qs8_vadd_minmax_ukernel__hvx_u128(
+    size_t batch,
+    const int8_t* input_a,
+    const int8_t* input_b,
+    int8_t* output,
+    const union xnn_qs8_add_minmax_params params[restrict XNN_MIN_ELEMENTS(1)]) XNN_OOB_READS
+{
+  assert(batch != 0);
+  assert(batch % sizeof(int8_t) == 0);
+  assert(input_a != NULL);
+  assert(input_b != NULL);
+  assert(output != NULL);
+
+  const HVX_Vector vbias = Q6_V_vsplat_R(*((int32_t *) &params->hvx.bias));
+  const HVX_Vector va_multiplier = Q6_V_vsplat_R(*((int32_t *) &params->hvx.a_multiplier));
+  const HVX_Vector vb_multiplier = Q6_V_vsplat_R(*((int32_t *) &params->hvx.b_multiplier));
+  const int32_t vshift = params->hvx.shift;
```

Comment:
Seems like this should not be prefixed `v` (it's scalar).

Reply:
Yes, this is fixed.